plugins/test/unit/plugins/distributors/export_distributor/test_distributor.py

```diff
@@ -62,10 +62,11 @@ def test_validate_config(self):
         # Clean up
         export_utils.validate_export_config = validate_config
 
+    @mock.patch.object(ISODistributor, 'ensure_all_units_downloaded')
```

Comment:
Just so you know, if you provide a value for the ``new`` kwarg here (it's the third arg), the instance is not passed as an argument to the decorated function.